### DISCUSSION TITLE
How to create cross-platform static or dynamic libraries for Andriod and ISO mobile platforms?

### DISCUSSION THREAD

**Discussion by smartware163** [microsoft/vscode-cpptools#11647] [Q&A]
### Feature Request

My questions are :
1. What compiler tools should we use?
2. Is there a project template for this purpose?
3. Are there any guide lines for this?

**Comment by guxiaobo** [comment]
Visual Studio Community Version has project templates for this, you can try custom cmake file projects to to this in VSCode.
